docs/architecture/grpc-for-wcf-developers/metadata.md

```diff
@@ -7,5 +7,53 @@ ms.date: 09/02/2019
 
 # Metadata
 
+"Metadata" refers to additional data that may be useful while processing requests and responses but is not part of the actual application data. Metadata might include authentication tokens, request identifiers and tags for monitoring purposes, or information about the data like the number of records in a dataset.
+
+It's possible to add generic key/value headers to WCF messages using an <xref:System.ServiceModel.OperationContextScope> and the <xref:System.ServiceModel.OperationContext.OutgoingMessageHeaders?displayProperty=nameWithType> property.
```

Comment:
There's also IncomingMessageHeaders. You can also get to headers via Message.Properties in places like message inspectors. Might be simpler to just reference the WCF MessageProperties class. If someone is using it, they should know what this is anyway.